### PR TITLE
(exa PR) 1119

### PR DESCRIPTION
**Pull request by cafkafk**
github.com/ogham/exa/pulls/1119

### CODE REVIEW

File: src/output/icons.rs [hunk 1]
```diff
@@ -48,22 +48,26 @@ pub fn iconify_style(style: Style) -> Style {
...
+        m.insert(".pam_environment", '\u{e615}'); // 
         m.insert(".rvm", '\u{e21e}'); // 
+        m.insert(".Trash", '\u{f1f8}'); // 
         m.insert(".vimrc", '\u{e62b}'); // 
         m.insert(".vscode", '\u{e70c}'); // 
+        m.insert(".xinitrc", '\u{f489}'); // 
+        m.insert(".xprofile", '\u{f489}'); // 
         m.insert(".zshrc", '\u{f489}'); // 
-        m.insert("Cargo.lock", '\u{e7a8}'); // 
+        m.insert("AUTHORS", '\u{f718}'); // 
```
Comment (cafkafk) [inline src/output/icons.rs:68]: This symbol doesn't render for me

File: src/output/icons.rs [hunk 2]
```diff
@@ -48,22 +48,26 @@ pub fn iconify_style(style: Style) -> Style {
...
         m.insert(".github", '\u{f408}'); // 
         m.insert(".gitignore", '\u{f1d3}'); // 
         m.insert(".gitmodules", '\u{f1d3}'); // 
+        m.insert(".pam_environment", '\u{e615}'); // 
         m.insert(".rvm", '\u{e21e}'); // 
+        m.insert(".Trash", '\u{f1f8}'); // 
         m.insert(".vimrc", '\u{e62b}'); // 
         m.insert(".vscode", '\u{e70c}'); // 
+        m.insert(".xinitrc", '\u{f489}'); // 
+        m.insert(".xprofile", '\u{f489}'); // 
```
Comment (cafkafk) [inline src/output/icons.rs:66]: I'm not sure about these

File: src/output/icons.rs [hunk 3]
```diff
@@ -48,22 +48,26 @@ pub fn iconify_style(style: Style) -> Style {
...
         m.insert(".zshrc", '\u{f489}'); // 
-        m.insert("Cargo.lock", '\u{e7a8}'); // 
+        m.insert("AUTHORS", '\u{f718}'); // 
         m.insert("bin", '\u{e5fc}'); // 
+        m.insert("Cargo.lock", '\u{e7a8}'); // 
```
Comment (cafkafk) [inline src/output/icons.rs:70]: this seems to be already present on main (at least it shows that icon when I run with `--icons`)

File: src/output/icons.rs [hunk 4]
```diff
@@ -81,12 +85,17 @@ lazy_static! {
         m.insert("hidden", '\u{f023}'); // 
         m.insert("include", '\u{e5fc}'); // 
         m.insert("lib", '\u{f121}'); // 
+        m.insert("LICENSE", '\u{f718}'); // 
```
Comment (cafkafk) [inline src/output/icons.rs:88]: This symbol doesn't render for me

File: src/output/icons.rs [hunk 5]
```diff
@@ -108,6 +117,7 @@ pub fn icon_for_file(file: &File<'_>) -> char {
     else if let Some(icon) = extensions.icon_file(file) { icon }
     else if let Some(ext) = file.ext.as_ref() {
         match ext.as_str() {
+            "a"             => '\u{f17c}', // 
```
Comment (cafkafk) [inline src/output/icons.rs:120]: Unsure what this even is

File: src/output/icons.rs [hunk 6]
```diff
@@ -118,6 +128,7 @@ pub fn icon_for_file(file: &File<'_>) -> char {
             "awk"           => '\u{f489}', // 
             "bash"          => '\u{f489}', // 
             "bash_history"  => '\u{f489}', // 
+            "bash_logout"   => '\u{f489}', // 
```
Comment (cafkafk) [inline src/output/icons.rs:131]: Not sure if this is optimal

**Review by cafkafk — COMMENTED** [review]
Not sure about these

File: src/output/icons.rs (hunk 1 above)
Comment (sbatial) [reply src/output/icons.rs:68]: they should render on the nerdfonts site: https://www.nerdfonts.com/cheat-sheet

I found it under 'document'
New name seems to be 'nf-md-file_document' (will be fixed with #9)

File: src/output/icons.rs (hunk 4 above)
Comment (sbatial) [reply src/output/icons.rs:88]: same document icon 'nf-md-file_document'

**Comment by sbatial** [comment]
I think it would make sense to maybe close this one and apply the proposed changes after #9 
Because as far as I can tell #9 is as good as ready now.

I would then go forward with reviewing it

**Pull request closed without merging**